sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobAsyncClient.java

```diff
@@ -151,14 +157,15 @@ boolean isRequiresEncryption() {
     EncryptedBlobAsyncClient(HttpPipeline pipeline, String url, BlobServiceVersion serviceVersion, String accountName,
         String containerName, String blobName, String snapshot, CpkInfo customerProvidedKey,
         EncryptionScope encryptionScope, AsyncKeyEncryptionKey key, String keyWrapAlgorithm, String versionId,
-        EncryptionVersion encryptionVersion, boolean requiresEncryption) {
+        EncryptionVersion encryptionVersion, boolean requiresEncryption, BlobClientSideEncryptionOptions encryptionOptions) {
```

Comment:
Can we name the parameter `clientSideEncryptionOptions` to make it consistent throughout this class?